File: Base64Sample.cs
using System;
using System.Text;

namespace Base64convert.Example
{
    public class Base64Sample {

        static void Main(string[] args)
        {
            string strB64Encoded = "SXQncyBsdW5jaCB0aW1l";
            byte[] data = Convert.FromBase64String(strB64Encoded);
            string strB64Decoded = UTF8Encoding.GetString(data);
            Console.WriteLine(strB64Decoded);

            string strB64Decoded = "It's lunch time";
            byte[] data = UTF8Encoding.GetBytes(strB64Decoded);
            string strB64Encoded = Convert.ToBase64String(data);
            Console.WriteLine(strB64Encoded);
        }
    }
}

File: Base64Sample.vb
Imports System
Imports System.Text

Namespace Base64convert.Example
    Public Class Base64Sample
        Public Shared Sub Main()
            Dim strB64Encoded As String = "SXQncyBsdW5jaCB0aW1l"
            Dim data As Byte() = Convert.FromBase64String(strB64Encoded)
            Dim strB64Decoded As String = UTF8Encoding.GetString(data)
            Console.WriteLine(strB64Decoded)

            Dim strB64Decoded As String = "It's lunch time"
            Dim data As Byte() = UTF8Encoding.GetBytes(strB64Decoded)
            ' convert the byte array to a Base64 string
            Dim strB64Encoded As String = Convert.ToBase64String(data)
            Console.WriteLine(strB64Encoded)
        End Sub
    End Class
End Namespace